intel_extension_for_transformers/neural_chat/pipeline/plugins/audio/utils/reduce_noise.py

```diff
@@ -52,7 +52,7 @@ def amp_to_db(x: torch.Tensor, eps=torch.finfo(torch.float64).eps, top_db=40) ->
 
 
 @torch.no_grad()
-def temperature_sigmoid(x: torch.Tensor, x0: float, temp_coeff: float) -> torch.Tensor:
```

Comment:
Can adding UT help to improve the coverage? Suggest not using pragma simply.

Reply:
Will check that.